PROJECT_TIERS.md

```diff
@@ -0,0 +1,63 @@
+# Project Tiers
+
+The MapLibre Organization maintains the map rendering frameworks MapLibre GL JS and MapLibre GL Native and fosters an ecosystem of plugins and tools around these libraries to produce the best maps in the world.
+
+The three project tiers Core, Supported, and Hosted describe qualitatively how donated resources are distributed among the projects.
+
+## Tiers
+
+### Core Project
+
+A project that the MapLibre Organization is committed to in a long-term way. A substantial amount of funding is spent on the project in the form of paid Maintainer time and paid Bounties to ensure its long-term success.
+
+### Supported Project
+
+A project that the MapLibre Organization can allocate funding to in the form of paid Maintainer time and paid Bounties. Supported Projects receive a substantially smaller amount of funding than Core Projects and the MapLibre Organization cannot guarantee any level of long-term support.
+
+### Hosted Project
+
+A project that the MapLibre Organization does not allocate any funding to. The project is hosted in the MapLibre GitHub organization to facilitate collaboration between the people who are interested in the project.
+
+## Process
+
+New projects start in the Hosted tier.
+
+If you would like to propose to change the tier of a project, please edit this file in a pull request.
+
+Changing the tier of a project requires approval by the MapLibre Governing Board, as the Board is responsible for all financial decisions.
+
+## List of Projects
+
+### Core
+
+* [maplibre-gl-js](https://github.com/maplibre/maplibre-gl-js)
+* [maplibre-gl-native](https://github.com/maplibre/maplibre-gl-native)
+
+### Supported
+
+* Currently none.
+
+### Hosted
+
+* [maplibre-gl-native-boost](https://github.com/maplibre/maplibre-gl-native-boost)
+* [mbtileserver-rs](https://github.com/maplibre/mbtileserver-rs)
+* [maplibre-plugins-android](https://github.com/maplibre/maplibre-plugins-android)
+* [demotiles](https://github.com/maplibre/demotiles)
+* [maplibre-gl-native-distribution](https://github.com/maplibre/maplibre-gl-native-distribution)
+* [maplibre-gl-directions](https://github.com/maplibre/maplibre-gl-directions)
+* [martin-landing-page](https://github.com/maplibre/martin-landing-page)
+* [maplibre-native-base](https://github.com/maplibre/maplibre-native-base)
```

Comment:
This is a part of the maplibre-gl-native C++ core and should be just as much in core as maplibre-gl-native. It'll likely be merged into the main repo at some point.